### PR TITLE
fix add AmolShelke2

### LINKED ISSUE #1034: AmolShelke2
## Profile Screenshot

github com_AmolShelke2

### PR DESCRIPTION
**Pull request by AmolShelke2**
resolves #1034

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/AmolShelke2.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/95171638/153740577-e9b68c79-cf30-403a-82eb-d1ce9cf7aada.png",
+  "issueId": 1034,
+  "name": "Amol Shelke",
+  "username": "AmolShelke2"
+}
```
Comment (Panquesito7) [inline profiles/AmolShelke2.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!, Thank you for your contribution :) :rocket:

**Comment by AmolShelke2** [comment]
> LGTM!, Thank you for your contribution :) 🚀
it's my pleasure

**Review by kabir0x17 — APPROVED** [review]
Nice Profile 👍

**Comment by AmolShelke2** [comment]
> Nice Profile 👍

Thank you

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**